iam_policy/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
```

Comment:
Move the terraform part to config.tf and refer https://github.com/variant-inc/lazy-terraform/blob/master/ec2/config.tf.

Reply:
Make sure that you use s3 backend and not local.